example/jupyterhub.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: tf-hub-lb
+  labels:
+    app: tf-hub
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 80
+    name: http
+    targetPort: 8000
+  selector:
+    app: tf-hub
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: tf-hub-0
```

Comment:
Why is this service named as it is?

Reply:
This is because the hub pods expect to connect back on tf-hub-0:8081. We could work around this and supply the fully qualified statefulset name, but that would need more configuration to be passed through. Chose this in the interest of simplicity.

Reply:
Why is it `-0`?